docs/tutorial.md

```diff
@@ -537,6 +543,66 @@ const myPromise = new Promise((resolve, reject) => {
 })
 ```
 
+### Loop over an object
```

Comment:
I'd add some warning to not use for..in.
Key arguments against for..in:
a) confusion with for..of
b) inherited properties

Reply:
I've added some info about the `for in` loop.